docker/importer/volume/surveillance-mediator/case-outcome/endpoint-resource-resolver-dhis.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "Covid19 Case Outcome DHIS2 Orchestrator",
+  "description": "Endpoint for mapping the Covid19 case outcome into DHIS format and then forwarding it to DHIS2",
+  "endpoint": {
+    "pattern": "/resource-resolver-dhis"
```

Comment:
The path doesn't match?

Reply:
@MattyJ007 Where is the required path documented?

Reply:
You added the path to the endpoint-handler file